packages/atomizer/src/rules.js

```diff
@@ -1343,6 +1343,38 @@ module.exports = [
     },
     /**
     ==================================================================
+    CONTAINERS
+    ==================================================================
+    // TODO: Discuss `n` , `nor`, `z`
+    */
+    {
+        type: 'pattern',
+        name: 'Container Name',
+        matcher: 'ContName',
+        allowParamToValue: true,
+        styles: {
+            'container-name': '$0',
+        },
+    },
+    {
+        type: 'pattern',
+        name: 'Container Type',
+        matcher: 'ContType',
+        allowParamToValue: false,
+        styles: {
+            'container-type': '$0',
+        },
+        arguments: [
+            {
+                is: 'inline-size',
+                n: 'none',
+                nor: 'normal',
+                s: 'size',
```

Comment:
we only have 1 other 'size' and that exists in same object as a `style`
```
            {
                n: 'none',
                st: 'strict',
                c: 'content',
                z: 'size',
                l: 'layout',
                s: 'style',
                p: 'paint',
            },
```

we could follow it and use `z` or keep `s` as we also have `is` for `inline-style`